File: Navigation/NavigationBottomOrRail/app/src/main/java/edu/cs4730/navigationbottomorrail/OneFragment.java
package edu.cs4730.navigationbottomorrail;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import edu.cs4730.navigationbottomorrail.databinding.FragmentOneBinding;

/**
 * A simple {@link Fragment} subclass.
 */
public class OneFragment extends Fragment {

    FragmentOneBinding binding;

    public OneFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        binding = FragmentOneBinding.inflate(inflater, container, false);
        return binding.getRoot();
    }

}

File: Basic/FragDemoSimple_kt/app/src/main/java/edu/cs4730/fragdemosimple_kt/oneFragment.kt
package edu.cs4730.fragdemosimple_kt

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import edu.cs4730.fragdemosimple_kt.databinding.FragmentOneBinding

/**
 * A simple fragment class.  this is the most basic code.  it does nothing.
 */
class oneFragment : Fragment() {
    private lateinit var binding: FragmentOneBinding

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        binding = FragmentOneBinding.inflate(inflater, container, false)
        return  binding.root
    }
}
